nesta/packages/quality_check/quality_calculations.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+import itertools
+
+def missing_values(data_frame):
+
+    """
+
+    """
+    return data_frame.isnull().sum().sort_values(ascending=False)
+
+def missing_value_column_count(data_frame):
+
+    bins_list = [i for i in range(100+1) if i%10 == 0]
+    out = pd.cut(data_frame.isnull().mean().apply(lambda x: round(100*x,5)), bins=bins_list, include_lowest=True)
+
+    return out.value_counts(sort=False)

```

Comment:
Might be a bit more readable to use numpy's `histogram` function

```suggestion
data_frame_missing = data_frame.isnull().sum() / data_frame.shape[0]
out = np.histogram(data_frame_missing, bins=range(0, 101, 10))
return out
```